<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calendrier</class>
 <widget class="QWidget" name="calendrier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion de mes agendas</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>20</width>
     <height>291</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>232</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Leelawadee UI</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
          <underline>true</underline>
          <strikeout>false</strikeout>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="text">
         <string>Informations de connexion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Leelawadee UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ID client</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ledt_clientID"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Leelawadee UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>client secret</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ledt_clientSecret"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_connexion">
        <property name="font">
         <font>
          <family>Leelawadee UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Connexion</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layoutCalendriers">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Leelawadee UI</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
          <underline>true</underline>
          <strikeout>false</strikeout>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="text">
         <string>Calendriers</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>311</width>
     <height>301</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>309</width>
      <height>299</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>311</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>311</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
